doc/configuration.rst

```diff
@@ -23,6 +41,17 @@ The config file is broken into sections, each controlling a different part of th
     [core]
     scheduler_host=luigi-host.mycompany.foo
 
+Example toml config:
+
+.. code:: python
```

Comment:
Because Pygments [doesn't have available lexerfor TOML](http://pygments.org/docs/lexers/). This is already contributed in some Pygments versions:
+ [pygments.rb](https://github.com/tmm1/pygments.rb/blob/b98c252f782ebb3fa6187ca4b46cddeb99985719/vendor/custom_lexers/github.py#L365)
+ [gitlab-pygments-rb](https://gitlab.com/gitlab-org/gitlab-pygments-rb/commit/d460ba68716937e59498f95fd34709d98990dea3)

But Pygments which used for travis checks doesn't have this lexer yet. I choose python as alternative lexer because toml syntax very similar to hybrid of python and ini :)

Reply:
And a TODO then please. :)